vllm/v1/worker/gpu_model_runner.py

```diff
@@ -2309,11 +2309,19 @@ def _bookkeeping_sync(
                 sampled_ids = [-1] if req_idx not in invalid_req_indices_set else None
             else:
                 sampled_ids = valid_sampled_token_ids[req_idx]
+
+            sampled_id_cnt: int = len(sampled_ids) if sampled_ids else 0
```

Comment:
for consistency with naming convention used elsewhere
```suggestion
            num_sampled_ids: int = len(sampled_ids) if sampled_ids else 0
```